zwave_js_server/const.py

```diff
@@ -3,6 +3,8 @@
 from typing import Dict, List
 
 MIN_SERVER_VERSION = "1.0.0-beta.2"
```

Comment:
Is this min version still ok? The add-on runs beta 4 now.

Reply:
I don't know. We are compatible with beta 2+ right ? It's not about the bugs, it's about the protocol.

Reply:
beta4 solved the missing keys in the state dumps so maybe require that as minimum

Reply:
Let's bump this separately.